core/src/main/java/org/infinispan/lifecycle/AbstractModuleLifecycle.java

```diff
@@ -1,59 +0,0 @@
-package org.infinispan.lifecycle;
-
-import org.infinispan.configuration.cache.Configuration;
-import org.infinispan.configuration.global.GlobalConfiguration;
-import org.infinispan.factories.ComponentRegistry;
-import org.infinispan.factories.GlobalComponentRegistry;
-
-
-/**
- * A default, no-op implementation of the {@link org.infinispan.lifecycle.ModuleLifecycle} interface, designed for easy
- * extension.
- *
- * @author Manik Surtani
- * @version 4.0
- * @deprecated since 9.1. Just implement {@link ModuleLifecycle} instead.
```

Comment:
Not sure about this, it wasn't deprecated in 9.0

Reply:
So in 10.0 we are only allowed to remove stuff deprecated in 9.0 but not 9.1+ ? So this and other cobwebs will have to be carried around until 11 ?